internal/app/machined/pkg/runtime/logging/circular.go

```diff
@@ -41,6 +42,8 @@ type CircularBufferLoggingManager struct {
 	sendersRW      sync.RWMutex
 	senders        []runtime.LogSender
 	sendersChanged chan struct{}
+
+	registeredLogs containers.ConcurrentMap[string, struct{}]
```

Comment:
why do we need another map given that we already have `buffers` ?

Reply:
Somehow I missed that we can range over buffers even tho I saw that that was a `sync.Map`. Thanks, we now use `buffers` instead.